types/zen-observable/esm.d.ts

```diff
@@ -0,0 +1,139 @@
+export function merge<A, B, C>(a: ObservableLike<A>, b: ObservableLike<B>, c: ObservableLike<C>): Observable<A | B | C>;
+export function merge<A, B, C, D>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+): Observable<A | B | C | D>;
+export function merge<A, B, C, D, E>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+): Observable<A | B | C | D | E>;
+export function merge<A, B, C, D, E, F>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+    f: ObservableLike<F>,
+): Observable<A | B | C | D | E | F>;
+export function merge<T>(...observables: Array<ObservableLike<T>>): Observable<T>;
+
+export function combineLatest(): Observable<never>;
+export function combineLatest<A>(a: ObservableLike<A>): Observable<[A]>;
+export function combineLatest<A, B>(a: ObservableLike<A>, b: ObservableLike<B>): Observable<[A, B]>;
+export function combineLatest<A, B, C>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+): Observable<[A, B, C]>;
+export function combineLatest<A, B, C, D>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+): Observable<[A, B, C, D]>;
+export function combineLatest<A, B, C, D, E>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+): Observable<[A, B, C, D, E]>;
+export function combineLatest<A, B, C, D, E, F>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+    f: ObservableLike<F>,
+): Observable<[A, B, C, D, E, F]>;
+export function combineLatest<T>(...observables: Array<ObservableLike<T>>): Observable<T[]>;
+
+export function zip(): Observable<never>;
+export function zip<A>(a: ObservableLike<A>): Observable<[A]>;
+export function zip<A, B>(a: ObservableLike<A>, b: ObservableLike<B>): Observable<[A, B]>;
+export function zip<A, B, C>(a: ObservableLike<A>, b: ObservableLike<B>, c: ObservableLike<C>): Observable<[A, B, C]>;
+export function zip<A, B, C, D>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+): Observable<[A, B, C, D]>;
+export function zip<A, B, C, D, E>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+): Observable<[A, B, C, D, E]>;
+export function zip<A, B, C, D, E, F>(
+    a: ObservableLike<A>,
+    b: ObservableLike<B>,
+    c: ObservableLike<C>,
+    d: ObservableLike<D>,
+    e: ObservableLike<E>,
+    f: ObservableLike<F>,
+): Observable<[A, B, C, D, E, F]>;
+export function zip<T>(...observables: Array<ObservableLike<T>>): Observable<T[]>;
```

Comment:
Maybe:
```typescript
import Observable from '.'

/**
 * @see https://github.com/zenparsing/zen-observable/blob/master/esm.js
 */
export default Observable
export { Observable }
```

Reply:
I agree, let's have something closer to the upstream package architecture. I had to make a few changes to do this, let me know what you think.